frontend/src/classes/TownsServiceClient.ts

```diff
@@ -1,6 +1,14 @@
 import axios, { AxiosInstance, AxiosResponse } from 'axios';
 import assert from 'assert';
 import { ServerPlayer } from './Player';
+import { NeighborStatus } from '../../../services/roomService/src/database/db';
```

Comment:
This is what's breaking your build

Reply:
Oh that makes a lot of sense, thanks professor!